frontend/packages/core/src/WorkflowLayout/index.tsx

```diff
@@ -90,22 +112,29 @@ const WorkflowLayout = ({
       !workflowPaths.find(path => !!matchPath({ path }, url))
   );
 
+  if (variant === "custom") {
+    return <>{children}</>;
+  }
+
   return (
     <LayoutContainer $variant={variant}>
       {!hideHeader && (
         <PageHeader $variant={variant}>
-          <Breadcrumbs entries={breadcrumbsEntries} />
-          <PageHeaderMainContainer>
-            {heading && (
-              <>
-                {React.isValidElement(heading) ? (
-                  heading
-                ) : (
-                  <Heading variant="h2">{heading}</Heading>
+          <PageHeaderBreadcrumbsWrapper $onlyBreadcrumbs={onlyBreadcrumbs}>
```

Comment:
🙋 what is `$onlyBreadcrumbs` used for?, in `PageHeaderBreadcrumbsWrapper`

Reply:
Good catch! It's a residue from a previous change, will remove it.